parsers/CA_AB.py

```diff
@@ -0,0 +1,162 @@
+        'production': {
+            'coal': total_net_generation['COAL'],
+            'gas': total_net_generation['GAS'],
+            'hydro': total_net_generation['HYDRO'],
+            'wind': total_net_generation['WIND'],
+            'unknown': total_net_generation['OTHER']
+        },
+        'capacity': {
+            'coal': maximum_capability['COAL'],
+            'gas': maximum_capability['GAS'],
+            'hydro': maximum_capability['HYDRO'],
+            'wind': maximum_capability['WIND'],
+            'unknown': maximum_capability['OTHER']
+        },
+        'source': 'ets.aeso.ca',
+    }
+
+
+def fetch_price(country_code='CA-AB', session=None):
+    """Requests the last known power price of a given country
+
+    Arguments:
+    country_code (optional) -- used in case a parser is able to fetch multiple countries
+    session (optional)      -- request session passed in order to re-use an existing session
+
+    Return:
+    A dictionary in the form:
+    {
+      'countryCode': 'FR',
+      'currency': EUR,
+      'datetime': '2017-01-01T00:00:00Z',
+      'price': 0.0,
+      'source': 'mysource.com'
+    }
+    """
+
+    r = session or requests.session()
+    url = 'http://ets.aeso.ca/ets_web/ip/Market/Reports/SMPriceReportServlet?contentType=html/'
+    response = r.get(url)
+    df_prices = pd.read_html(response.text, match='Price', index_col=0, header=0)
+    prices = df_prices[1]
+
+    data = {}
+
+    for rowIndex, row in prices.iterrows():
+        price = row['Price ($)']
+        if (isfloat(price)):
+            hours = int(rowIndex.split(' ')[1]) - 1
+            data[rowIndex] = {
+                'datetime': arrow.get(rowIndex, 'MM/DD/YYYY').replace(hours=hours, tzinfo=timezone).datetime,
+                'countryCode': country_code,
+                'currency': 'CAD',
+                'source': 'ets.aeso.ca',
+                'price': float(price),
+            }
+
+    return [data[k] for k in sorted(data.keys())]
+
+
+def fetch_exchange(country_code1='CA-AB', country_code2='CA-BC', session=None):
+    """Requests the last known power exchange (in MW) between two countries
+
+    Arguments:
+    country_code (optional) -- used in case a parser is able to fetch multiple countries
+    session (optional)      -- request session passed in order to re-use an existing session
+
+    Return:
+    A dictionary in the form:
+    {
+      'sortedCountryCodes': 'DK->NO',
+      'datetime': '2017-01-01T00:00:00Z',
+      'netFlow': 0.0,
+      'source': 'mysource.com'
+    }
+    """
+
+    r = session or requests.session()
+    url = 'http://ets.aeso.ca/ets_web/ip/Market/Reports/CSDReportServlet'
+    response = r.get(url)
+    df_exchanges = pd.read_html(response.text, match='INTERCHANGE', skiprows=0, index_col=0)
+    
+    flows = {
+        'CA-AB->CA-BC': df_exchanges[1][1]['British Columbia'],
+        'CA-AB->CA-SK': df_exchanges[1][1]['Saskatchewan'],
+        'CA-AB->US-MT': df_exchanges[1][1]['Montana']
```

Comment:
For now we only have US and not US-MT (we don't break down into US states yet).
Can you just replace US-MT with US?